src/lib/bootloader/autoyast_converter.rb

```diff
@@ -235,7 +236,8 @@ def export_grub2efi(res, bootloader)
       # only for grub2, not for others
       GRUB2_BOOLEAN_MAPPING = {
         "secure_boot"  => :secure_boot,
-        "trusted_grub" => :trusted_boot
+        "trusted_grub" => :trusted_boot,
+        "update_nvram"  => :update_nvram,
```

Comment:
so update nvram can be done also for non-EFI systems?

Reply:
it applies to EFI and PPC